documentation/modules/adding-source-provider.adoc

```diff
@@ -78,6 +80,7 @@ ifdef::vmware[]
 . Specify the following fields:
 
 * *Provider resource name*: Name of the source provider.
+* *Endpoint type*: Select the vSphere provider endpoint type. Options: *vSphere* or *ESXi*.
 * *URL*: URL of the SDK endpoint of the vCenter on which the source VM is mounted. Ensure that the URL includes the `sdk` path, usually `/sdk`. For example, `https://vCenter-host-example.com/sdk`. If a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate.
 * *VDDK init image*: `VDDKInitImage` path. It is strongly recommended to create a VDDK init image to accelerate migrations. For more information, see xref:../master.adoc#creating-vddk-image_mtv[Creating a VDDK image].
 * *Username*: vCenter user. For example, `user@vsphere.local`.
```

Comment:
vCenter and ESXi may have different user name and password, so this should not make users think they need to use there vCenter credentials even if they connect to ESXi, on ESXi they should use the ESXi user+password and on vCenter they should use the vCenter user+password,  , @ahadas please keep me honest here